omod/src/main/java/org/openmrs/module/idgen/rest/resource/IdentifierSourceResource.java

```diff
@@ -0,0 +1,297 @@
+            }
+            if(errors.toString().length() > 2) {
+                throw new ConversionException("The values of the following inputs are missing or invalid: " + errors.toString());
+            }
+
+            identifierSource.setIdentifierType(patientIdentifierType);
+            identifierSource.setName(name);
+            identifierSource.setUrl(url);
+            identifierSource.setDescription(description);
+
+            savedIdentifierSource =  save(identifierSource);
+
+        }
+        else if(identifierSourceType.equals("IdentifierPool")){
+
+            IdentifierPool identifierSource = new IdentifierPool();
+            String batchSize = validateInput(postBody.get("batchSize"));
+            String minPoolSize = validateInput(postBody.get("minPoolSize"));
+            String sourceUuid = validateInput(postBody.get("sourceUuid"));
+            String sequential = validateInput(postBody.get("sequential"));
+            String refillWithScheduledTask = validateInput(postBody.get("refillWithScheduledTask"));
+            IdentifierSource poolSource = Context.getService(IdentifierSourceService.class).getIdentifierSourceByUuid(sourceUuid);
+
+            if (!StringUtils.isBlank(batchSize)) {
+                if(StringUtils.isNumeric(batchSize)) {
+                    identifierSource.setBatchSize(Integer.parseInt(batchSize));
+                }else {
+                    errors.add("batch size");
+                }
+            }
+            if (!StringUtils.isBlank(minPoolSize)) {
+                if(StringUtils.isNumeric(minPoolSize)) {
+                    identifierSource.setMinPoolSize(Integer.parseInt(minPoolSize));
+                    Set<PooledIdentifier> pooledIdentifiers = new LinkedHashSet<PooledIdentifier>();
+                    for(Integer counter = 0; counter < Integer.parseInt(minPoolSize); counter++) {
+                            pooledIdentifiers.add(new PooledIdentifier(identifierSource, "ABC0" + counter));
+                    }
+                    identifierSource.setIdentifiers(pooledIdentifiers);
+                }else {
+                    errors.add("min pool size");
+                }
+            }
+            if (!StringUtils.isBlank(sequential)) {
+                if(sequential.toLowerCase().equals("true") || sequential.toLowerCase().equals("false")) {
+                    identifierSource.setSequential(Boolean.parseBoolean(sequential));
+                }else {
+                    errors.add("sequential");
+                }
+            }
+            if (!StringUtils.isBlank(refillWithScheduledTask)) {
+                if(refillWithScheduledTask.toLowerCase().equals("true") || refillWithScheduledTask.toLowerCase().equals("false")) {
+                    identifierSource.setSequential(Boolean.parseBoolean(refillWithScheduledTask));
+                }else {
+                    errors.add("refill");
+                }
+            }
+            if(errors.toString().length() > 2) {
```

Comment:
Same question here as before for ignoring cases for one or two errors.

Reply:
I have answered below.